paperqa/agents/__init__.py

```diff
@@ -79,16 +79,16 @@ def configure_cli_logging(verbosity: int = 0) -> None:
 
     rich_handler.setFormatter(logging.Formatter("%(message)s", datefmt="[%X]"))
 
-    module_logger = logging.getLogger(__name__.split(".")[0])
+    module_logger = logging.getLogger(__name__.split(".", maxsplit=1)[0])
```

Comment:
I don't think we want maxsplit here -- like `paperqa.agents.search` -- we want two splits there.

Reply:
We throw away everything beyond `[0]` so I think `maxsplit` would be a slight improvement